perma_web/tasks/dev.py

```diff
@@ -1162,21 +1165,41 @@ def benchmark_wacz_conversion(ctx, benchmark_log, source_csv=None, single_warc=N
     # for a rough estimate of how long all jobs took to be processed by celery
     logger.info(f"Benchmark CSV was created at: {datetime.now()}")
 
+    base_links_query = Link.objects.filter(
+            is_private=False,
+            is_unlisted=False,
+            cached_can_play_back=True
+    ).values_list('guid', flat=True)
+
+    links = Link.objects.none()
+
     if source_csv:
         sample_data_guids = os.path.abspath(source_csv)
         with open(sample_data_guids, mode='r') as file:
             csv_file = csv.reader(file)
             for line in csv_file:
                 convert_warc_to_wacz.delay(line[0], log_file)
-    elif single_warc:
+        return
+
+    if single_warc:
         convert_warc_to_wacz.delay(single_warc.split('.')[0], log_file)
-    else:
-        links = Link.objects.filter(
-            is_private=False,
-            is_unlisted=False,
-            cached_can_play_back=True
-        ).values_list('guid', flat=True).order_by('guid')[:batch_size]
+        return
 
-        for link in links.iterator():
-            convert_warc_to_wacz.delay(link, log_file)
+    if batch_range:
+        batch_range_start, batch_range_end = batch_range.split(':')
```

Comment:
This is completely minor, but you could do
```suggestion
        batch_range_start, batch_range_end = (int(i) for i in batch_range.split(':'))
```
and lose the next two lines, or make it a list comprehension if the pseudo-tuple comprehension is too weird.

Reply:
Thank you @bensteinberg, I think I will leave as is as I find it simpler in terms of readability :)